docs/creating-theme.md

```diff
@@ -1,11 +1,222 @@
+- `templates` containing the the overriden templates.
+
+!!! note
+    This is a proposal layout for a standalone theme.
+    As long as the theme package has the proper layout (`info.json`, `__init__.py`...),
+    it can be wherever you want if you properly expose it in your `setup.py` file.
+
+### `setup.py`
+
+The `setup.py` is a classic python `setup.py` file.
+The only requirements is that you properly expose the udata theme packaging
+as `udata.themes` entrypoint:
+
+```python
+setup(
+    '...'
+    entry_points={
+        'udata.themes': [
+            'any-identifier = canonical.theme.package',
+        ]
+    },
+    '...'
+)
+```
+
+### `info.json`
+
+The ``info.json`` looks like this:
+
+```json
+{
+    "application": "udata",
+    "identifier": "my-theme",
+    "name": "My awesome theme",
+    "author": "Me",
+    "description": "An awesome theme for udata",
+    "website": "http://awesome.opendata.tem",
+    "license": "AGPL",
+    "version": "0.1.0",
+    "doctype": "html5"
+}
+```
+The `application` and the `doctype` attributes needs to have specific values, respectively `udata` and `html5`.
+The `identifier` attribute is important: this is the value you will be using in `udata.cfg` to use your theme
+(the `THEME` parameter).
+Any other attribute can have any value, this is only metadata.
+
+### Static assets
+
+The `static` should contains at least 2 files:
+
+- `theme.css` which is loaded automatically on every page
+- `admin.css` which is loaded automatically on on the amdin page
+
+Some images are required and used everywhere in the application:
+
+- `img/flags` contains the language switcher flags in the form `{language code}.png` (ex. `en.png`).
+- `img/placeholders` contains default images for:
+    - users without avatar (`user.png`)
+    - organizations without logo (`organization.png`)
+    - reuses without image (`reuse.png`)
+    - territory without logo or flag (`territory.png`)
+
+Then your free to add any static assets reuired by your theme
 
+### Overriding templates
+
+You can override or extend any default Jinja template.
+You only need to put the templates with the same name into the `templates` folder.
+
+To extend a template and change some details, just extend the base template and override the block you want to change:
+
+```html+jinja
+{% extends "raw.html" %}
+
+{% block raw_head %}
+{{ super() }}
+<link rel="shortcut icon" href="{{ theme_static('img/favicon.png') }}">
+{% endblock %}
+```
+You can reference static assets from you theme with the `theme_static` global function.
+
+!!! note
+    Don't forget to call `{{ super() }}` to includes the original block.
+    See the [Jinja documentation][] for advanced usage.
+
+You can also rewrite entirely the template, but don't forget you need to have the proper blocks
+for template inheritance and use the proper context variables.
+
+### Hooks
+
+You're theme can also customize some behavior be using hooks in you ``__init__.py``.
```

Comment:
YouR + behaviors + bY + youR => `Your theme can also customize some behaviors by using hooks in your ``__init__.py``.`